src/brod_group_subscriber_v2.erl

```diff
@@ -415,6 +416,7 @@ terminate_all_workers(Workers) ->
 terminate_worker(WorkerPid) ->
   case is_process_alive(WorkerPid) of
     true ->
+      unlink(WorkerPid),
       brod_topic_subscriber:stop(WorkerPid);
```

Comment:
This `unlink` call could be moved to `brod_topic_subscriber:stop` to fix this issue in a generic way.

Reply:
Thanks for the review! Sure, `unlink` could be moved to `stop`, but I'm not sure it is a desirable property in a general case - it is a little less explicit and in some cases the user might not even want that - for example, some other `brod_topic_subscriber`'s user may want to handle a crash during `brod_topic_subscriber` termination and `unlink`'ing would prevent that. Another argument I have is that since `brod_group_subscriber_v2` traps exits, it has to make sure it handles them correctly so it shouldn't probably depend on a `stop`s side effect for that. For consistency sake if this change is made then it would make sense to do the same in the other modules' `stop`'s too by the way.

But I might be wrong here, it's your call :)

Reply:
I think stopping of all processes should be done uniformly, but now it's not the case for different reasons. I guess we can fix it later.